.pipelines/e2e-windows.yaml

```diff
@@ -41,6 +41,8 @@ stages:
       windowsImage: 2019-containerd
       windowsOSSKU: Windows2019
       windowsVMSize: Standard_D2s_v3
+      windowsDistro: aks-windows-2019-containerd
+      windowsNodeImageVersion: AKSWindows-2019-containerd-test-2023.02.07
```

Comment:
Why is the value `2023.02.07`?

Reply:
Currently we use the test image referred in Line 139 of e2e/windows/e2e-scenario.sh and it was built by ourselves on 2023/02/07. 

In the future, after moving our vhd sub to ame sub, we can use prod image and sync with the latest image.